### DISCUSSION TITLE
So, with the current DSStats there is no real history, lets give it hisotory

### DISCUSSION THREAD

**Discussion by TheWitness** [Cacti/cacti#5228] [Ideas]
So, not much of a discussion, but more a statement that in the 1.2.x versions of DSStats, they were missing some important metrics to help in making more interesting Graph Tree's and dashboards.

Those things that were missing were some additional statistical data like Median, 95th percentile, Standard Deviation and a few others.  Those gaps are already closed, but now, what about every day, week, month, and year, creating partitions that allow you to say, within milliseconds, which were the ten top interfaces last week.  What hosts had the highest load average yesterday.  What was the total bandwidth of the switch over the last 4 weeks.

Yup for that we need more DSStats tables.  That's what I plan to do.  Just thought I would drop this out there for others to comment on.

**Comment by bmfmancini** [comment]
Yes! Getting asked more and more for these reports would be a great to see
